### PR TITLE
Optimisation of code

### PR DESCRIPTION
**Pull request by PriyaaIyer**
Fixes #
- Code optimised for assignment 1,2 and 3

## Proposed Changes

-
-
-

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/53"></a>

### CODE REVIEW

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 1]
```diff
@@ -1,49 +1,60 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
+   <meta charset="UTF-8">
+   <title>Page Title</title>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:6]: Mention a proper page title

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 2]
```diff
@@ -1,49 +1,60 @@
...
 </head>
+
 <body>
 
-<h1>Merlo J. Pusey</h1>
-<hr>
-<p><em>From Wikipedia, the free encyclopedia</em></p>
+   <h1>Merlo J. Pusey</h1>
+   <hr>
+   <p><em>From Wikipedia, the free encyclopedia</em></p>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:13]: Use `cite` instead of `em`.

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 3]
```diff
@@ -1,49 +1,60 @@
...
+
+   <img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" width="250" , height="200" />
 
-<img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" width="250" , height="200" />
+   <p><b>Merlo John Pusey</b>(February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington, D.C.)[1][2] was an
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:17]: Use `strong` instead of `b`

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 4]
```diff
@@ -1,49 +1,60 @@
...
-        
-       <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946, continuing to contribute occasional pieces until about two years before his death.
-        <br></p>
-        <p><b>Contents</b></p>
-        <p><a href="Books.html">1. Books</a><br>
-           <a href="References.html">2. References</a></p>
+   <p>Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints
+      University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the college
+      newspaper. He later became
+      a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]<br></p>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:25]:
```suggestion
      a reporter and assistant city editor at The Deseret News in Salt Lake City.<sup>[3]</sup><br></p>
```

File: Priyaa/HTML Basics Remake/Assignment-2.html [hunk 5]
```diff
@@ -1,45 +1,58 @@
 <!DOCTYPE html>
 <html>
+
 <head>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-2.html:4]: The metadata is missing. Use VS Code's autofill to make a basic HTML template (AKA Boilerplate).

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 6]
```diff
@@ -2,90 +2,61 @@
 <html>
 
 <head>
-    <title>Assignment-Registration Form</title>
+        <title>Assignment-Registration Form</title>
 </head>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:6]: The metadata is missing.

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 7]
```diff
@@ -2,90 +2,61 @@
...
 
-    <form id="registration_form" method="post">
-        <label for="first name"><b>First name:</b></label>
-        <input type="text" 
-                name="firstname" 
-                placeholder="First Name" 
-                autocomplete="given-name" 
-        required>
+        <form id="registration_form" method="post">
+                <label for="first name"><b>First name:</b></label>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:17]: Use `strong` instead of `b`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 8]
```diff
@@ -2,90 +2,61 @@
...
+                <input type="text" name="firstname" placeholder="First Name" autocomplete="given-name" required>
 
-        <label for="last name"><b>Last name:</b></label>
-        <input type="text" 
-                name="lastname" 
-                placeholder="Last Name" 
-                autocomplete="family-name" 
-        required>
-        <br><br>
+                <label for="last name"><b>Last name:</b></label>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:20]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 9]
```diff
@@ -2,90 +2,61 @@
...
 
 
-        <b>Gender:</b>
-        <label for="male">Male</label>
-        <input type="radio" 
-                name="gender" 
-                id="male" 
-                value="male" 
-        required>
+                <b>Gender:</b>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:26]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 10]
```diff
@@ -2,90 +2,61 @@
...
 
 
 
-        <label for="E-mail"><b>E-mail</b></label>
-        <input type="email"
-                 name="email" 
-                 placeholder="E-mail" 
-                 autocomplete="email" 
-        required>
+                <label for="E-mail"><b>E-mail</b></label>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:39]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 11]
```diff
@@ -2,90 +2,61 @@
...
 
-        <label for="Password"><b>Password</b></label>
-        <input type="password" 
-                name="password" 
-                placeholder="Password" 
-                minlength="5" 
-                maxlength="15" 
-                autocomplete="new-password" 
-        required>
+                <label for="Password"><b>Password</b></label>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:42]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 12]
```diff
@@ -2,90 +2,61 @@
...
+                <input type="password" name="password" placeholder="Password" minlength="5" maxlength="15"
+                        autocomplete="new-password" required>
 
-        <label for="Birthday"><b>Birthday</b></label>
-        <input type="date" 
-                name="birthday" 
-                autocomplete="bday" 
-                required>
-        <br><br>
+                <label for="Birthday"><b>Birthday</b></label>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:46]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 13]
```diff
@@ -2,90 +2,61 @@
...
+                <input type="date" name="birthday" autocomplete="bday" required>
+                <br><br>
 
 
 
-        <label for="terms and conditions"><b>I agree to the terms and conditions</b></label><input type="checkbox" required>
-        <br><br>
-        <input type="submit">
-    </form>
+                <label for="terms and conditions"><b>I agree to the terms and conditions</b></label><input
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html:52]: Use `strong`

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 14]
```diff
@@ -1,49 +1,60 @@
...
 
-           <h2>Books</h2>
-           <hr>
-           <ul>
-              <li>The Supreme Court Crisis(Macmillan, 1937)</li>
-              <li>Big Government: Can We Control It? (Harper, 1945)</li> 
-              <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
-               <li> Eisenhower, the President (Macmillan, 1956)</li></p>
-            </ul>
+   <p><strike>Pusey</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946,
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:27]:
```suggestion
   <p><strike>Pussy</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in 1946,
```

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 15]
```diff
@@ -1,49 +1,60 @@
...
+      continuing to contribute occasional pieces until about two years before his death.
+      <br></p>
+   <p><b>Contents</b></p>
+   <p><a href="Books.html">1. Books</a><br>
+      <a href="References.html">2. References</a></p>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:32]: Use ordered list

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 16]
```diff
@@ -1,49 +1,60 @@
...
 
+   <h2>Books</h2>
+   <hr>
+   <ul>
+      <li>The Supreme Court Crisis(Macmillan, 1937)</li>
+      <li>Big Government: Can We Control It? (Harper, 1945)</li>
+      <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
+      <li> Eisenhower, the President (Macmillan, 1956)</li>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:40]: The content in brackets is in italics. Use `em`

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 17]
```diff
@@ -1,49 +1,60 @@
...
+
+   <h2>References</h2>
+   <hr>
+   <ul>
+      <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286. Retrieved
+         2016-03-24.</li>
+      <li>Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York Times. ISSN
+         0362-4331. Retrieved 2016-03-24.</li>
+      <li>Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li>
+   </ul>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:53]: Use `div` with `id` to link this section with `a` tag you used in the content section.

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 18]
```diff
@@ -1,49 +1,60 @@
...
 
+   <h2>Books</h2>
+   <hr>
+   <ul>
+      <li>The Supreme Court Crisis(Macmillan, 1937)</li>
+      <li>Big Government: Can We Control It? (Harper, 1945)</li>
+      <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li>
+      <li> Eisenhower, the President (Macmillan, 1956)</li>
+      </p>
+   </ul>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html:42]: Use `div` with `id` to link this section with `a` tag you used in the content section.

File: Priyaa/HTML Basics Remake/Assignment-2.html [hunk 19]
```diff
@@ -1,45 +1,58 @@
...
-    <td><a href="http://pokemon.wikia.com/wiki/Wartortle.html/">Wartortle</a></td>
-  </tr>
-</table>
+  <table style="width:25%">
+    <tr>
+      <th>Image</th>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Evolves Into</th>
+    </tr>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-2.html:30]: Wrap this section in `thead` tag

File: Priyaa/HTML Basics Remake/Assignment-2.html [hunk 20]
```diff
@@ -1,45 +1,58 @@
...
+      <td><a href="http://pokemon.wikia.com/wiki/Charmelon.html/">Charmelon</a></td>
+    </tr>
+    <tr>
+      <td><img
+          src="http://img1.wikia.nocookie.net/__cb20140328191525/pokemon/images/thumb/3/39/007Squirtle.png/200px-007Squirtle.png">
+      </td>
+      <td>Squirtle</td>
+      <td>Water</td>
+      <td><a href="http://pokemon.wikia.com/wiki/Wartortle.html/">Wartortle</a></td>
+    </tr>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-2.html:54]: Wrap this section in `tbody` tag

**Review by VinayakTekade — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
Follow this:

- Delete your fork from your GitHub profile and your computer.
- Fork the repository as the history of the repository has changed.
- Clone your fork to your computer, make a new branch and start working.
- Send in a pull request from your new branch. Once merged, delete that branch or make sure it stays in sync. Make sure your master branch is always in sync as well.

Also, make the changes requested by @VinayakTekade

**Pull request closed without merging**